rosdoc2/main.py

```diff
@@ -19,6 +19,8 @@
 from osrf_pycommon.cli_utils.verb_pattern import list_verbs
 from osrf_pycommon.cli_utils.verb_pattern import split_arguments_by_verb
 
+from . import __version__ as VERSION
```

Comment:
As of 3.8 importlib.metadata provides a way to do this instead of the standalone file: https://docs.python.org/3/library/importlib.metadata.html#distribution-versions  I think that it would be a better way for maintainability and consistency.

Reply:
As an example: https://stackoverflow.com/questions/15405636/pythons-argparse-to-show-programs-version-with-prog-and-version-string-formatt